api/v1/types_util.go

```diff
@@ -520,6 +520,10 @@ func (ispn *Infinispan) AddLabelsForPods(uMap map[string]string) {
 	addLabelsFor(ispn, PodTargetLabels, uMap)
 }
 
+func (ispn *Infinispan) AddVersionLabelForPods(uMap map[string]string) {
+	uMap["infinispan_version"] = strings.Split(consts.DefaultImageName, ":")[1]
```

Comment:
label name should conform k8s recommended rules https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/ and could be like
`infinispan.org/version`

Reply:
That's true @dmvolod but unfortunately none of our existing labels follow this strategy atm. I'll create a GH issue to address this in a different PR.

Reply:
The additional labels can be handled in another PR https://github.com/infinispan/infinispan-operator/issues/597

Reply:
coordinator is living out his last days and will be removed in #1207 
Others need to conform, but for Pod labels selectors it's a bit dangerous for Rolling Upgrade, as controllers are working in non determined sequence (the same as golang routines) service could be updated early than Pod labels and kube-dns could disable network connectivity for some moments due to inconsistent labels matching